rathemtn/Yilbit_Gerbigo.lua

```diff
@@ -0,0 +1,50 @@
+-- Yilbit Gerbigo
+-- Marr's Blessing
+-- items: 55035, 55036
+function event_say(e)
+	local fac = e.other:GetFaction(e.self);
+	
+	if(e.other:Race() ==  "Froglok" or fac < 5) then
+		if(e.message:findi("hail")) then
+			e.self:Say("Salutations. Come sit with me and have a drink. I am just going over some [" .. eq.say_link("business") .. "] right now.");
```

Comment:
No need for any eq.say_link calls in Say types, The source code will autocovert [word] to saylinks (just not in Emotes)

Reply:
Thanks for the tips -- I'll update the lua file with these suggestions and submit another pull request. :)

Reply:
Sounds good, just commiting to the same local branch will add it to the current PR

Reply:
Heh, I missed one of the say_link calls -- sorry about that. Going to clean it up and commit again.

Reply:
lgtm - Thank you for the PR